Update doc for implementing cards

mmeldo opened the pull request.

janosikm commented on docs/implementing-cards.md:
```diff
@@ -1,47 +1,95 @@
+In case you are using snippet, just type the snippet prefix (e.g. `dudecard`) and VSCode should give you suggestion. If not, press Ctrl+space.
 
-The card class should have its `code` property set to the unique card identifier for that card. You can find these combining the 2-digit pack number (01) and 3-digit card number (036), or on [thronesdb.com](https://thronesdb.com/) by looking at the URL for the specific card.
+The card class should have its `code` property set to the unique card identifier for that card. You can find these in `/server/game/cards/cards_summary.csv`, or on [dtdb.co](https://dtdb.co/) by looking at the URL for the specific card. Or by combining the 2-digit pack number (01) and 3-digit card number (035).
```
cards_summary.csv should be updated to store all packs and cards now there are only packs till There Comes a  Reckoning

mmeldo replied:
Yes it should :)

mmeldo replied:
will be done separately as latest packs are not yet processed (they are not in database)

mmeldo commented on docs/implementing-cards.md:
```diff
@@ -1,47 +1,95 @@
-##### 1. Create a file named after the card.
+#### 1. Create a file named after the card.
 
 Cards are organized under the `/server/game/cards` directory by grouping them by "\<cycle number>-\<pack code>". Each word in the file name should be capitalized.
```
Directories for packs are already created.

mmeldo replied:
it turned out empty directories are not pushed to GitHub so I created empty file in each of the packs. It will be pushed to the PR later.

mmeldo commented:
New section for putting cads into play was added.

mmeldo commented:
Updated `Getting started` with the fourth step that suggest what chapter (action type) should be used based on card text. Also added `General Information` with additional methods for rank modification, GR manipulation and others.

mmeldo commented:
Can you please check the doc once more?

mmeldo commented:
Fixed some typos.
